Pool_Properties_Design.md

```diff
@@ -0,0 +1,213 @@
+# Design to Define Storage Pool Properties
+
+**Status**: Completed
+
+**Version**: Alpha
+
+**Author(s)**: Leon Wang (@leonwanghui)
+
+## Summary
+
+This proposal is designed for standardizing the properties definition of storage pool. This design is also part of referenced implementation of LineOfService defined in [Swordfish](https://www.snia.org/sites/default/files/SMI/swordfish/V105/Swordfish_v1.0.5_Specification.pdf) specification (v1.0.5), which promises to provide a unified approach for the management of storage and servers in hyperscale and cloud infrastructure environments.
```

Comment:
Not really related to the design discussion here, but just want to point out that word wrapping is very dependent on the editor used to view this file. So it might be nice to put in some line breaks for some of these (80 characters/line is a generally good rule of thumb) so plain text editors don't need to scroll so far to the right to read everything.